spec/features/training_tool_spec.rb

```diff
@@ -264,6 +264,11 @@ def check_slide_contents(slide, slide_number, slide_count)
 end
 
 def proceed_to_next_slide
+  if page.has_selector?('.alert-box-container', wait: 10)
+    within('.alert-box-container') do
+      page.execute_script("$('.alert-box-container').hide()")
```

Comment:
I'd rather avoid introducing new cases of `execute_script`. Can this just click 'CLOSE' instead?

Reply:
Done! I've removed additional script and used .click for achieving the same.